src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2565,6 +2565,9 @@ namespace DataHeatBalance {
         Real64 CO2Rate;
         Real64 GCRate;
 
+        Real64 SumTinMinusTSup;  // Numerator for zone-level sensible heat index (SHI)
```

Comment:
Data in local arrays made member variables here.

Reply:
:+1: